### PR TITLE
Translate to Spanish

### LINKED ISSUE #75: Española translation following CONTRIBUTING.md
# Feature Request

## Describe the solution you'd like

Following the `CONTRIBUTING.md` section on translations provide a Española translation.

- [x] `README.md` -> `.github/lang/README.es.md`
- [x] `CONTRIBUTING.md` -> `.github/lang/CONTRIBUTING.es.md`
- [ ] `lang/en.js` -> `lang/es.js`
- [ ] `content/en/index.md` -> `content/es/index.md`
- [ ] `content/en/details.md` -> `content/es/details.md`

### PR DESCRIPTION
**Pull request by alvaroartano**
# Pull Request

## Description:

Translated the remaining files

## Related Issue:

Closes #75

### CODE REVIEW

**Review by lukeocodes — CHANGES_REQUESTED** [review]
There were some missing strings in the English translation. Can you rebase and then check the en.js file for the new properties?

**Comment by alvaroartano** [comment]
i think it's all right (that's what it seems)

**Comment by lukeocodes** [comment]
It's missing the nav strings... They just got missed at the start, they've now been added. See the English and German translation files. If they're the same words, please add them to the `es.js` language file anyway.

**Review by alvaroartano — COMMENTED** [review]
done. all fixed

File: lang/es.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+   * Strings used in /components/**.vue files
+   */
+  components: {
+    announcement: {
+      button: 'Sitio no oficial',
+      text:
+        'Este sitio no es oficial. Clica aquí para ir a la web oficial del Hacktoberfest.',
+    },
+    
+      navigation: {
```
Comment (lukeocodes) [inline lang/es.js:49]:
```suggestion
    navigation: {
```

File: lang/es.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      title: 'Participan en <strong>Hacktoberfest?</strong>',
+      error: 'Error',
+      retry: 'Quieres probarlo de nuevo?',
+      success:
+        'Este proyecto participa en el <strong>Hacktoberfest</strong> por el momento.',
+      failure:
+        'Estre proyecto no forma parte del <strong>Hacktoberfest</strong> por el momento.',
+      results_are_in: 'El resultado es',
+      cta: 'Participan?',
+    },
```
Comment (lukeocodes) [inline lang/es.js:65]:
```suggestion
      cta: 'Participan?',
      noRepoFound: 'No previous repo checks found... yet!',
    },
```

**Review by lukeocodes — CHANGES_REQUESTED** [review]
Just two little bits :)

**Review by alvaroartano — COMMENTED** [review]
Fixed

**Pull request merged**